projects/managed-hubs/support.md

```diff
@@ -179,19 +179,38 @@ The current iteration of the workflow states each step and who should be respons
 
 When a new ticket lands in Freshdesk under the support group and it is not an incident, we aim to respond within 24 working hours with a suggested next action. The next steps should be followed when resolving a ticket:
 
-1. `Who: Support steward`
+1. `Who:Support steward`
 
-   **First 24h initial ticket evaluation**. In the first 24h a support ticket was opened, you should do an initial evaluation of the ticket and ask the {term}`Community Representative` about any additional information you may need.
+   First, we determine if the person *initiating* the support ticket is *authorized* to do actually do so. While we may interact with many folks
+   from a community during resolution of a ticket, we constrain who can *initiate* a ticket to {term}`Community Representative`s only. This prevents our
+   support staff from being overwhelmed by tickets that need to be handled elsewhere. If the person *initiating* the ticket is not a community
+   representative, the support steward should cc the community representatives, and ask for approval. The support steward *may* choose to use the following
+   email template:
```

Comment:
```suggestion
   email template also available as a [Canned Response](https://2i2c.freshdesk.com/a/admin/canned_responses/folders):
```

I added this into our canned responses on Freshdesk as well so it's really easy to find.